internal/standby/standby.go

```diff
@@ -119,17 +125,21 @@ func (s *Service) checkForOutage(currentTime time.Time) {
 		if s.InCommandMode() {
 			s.logger.Info("Commands resumed after outage", "time since last command", timeSinceLastCmd)
 			s.setMode(StandbyMode)
+			s.logHandler.Append("Resumed standby mode", map[string]string{"timeSinceLastCmd": timeSinceLastCmd.String()})
 		}
 		return
 	}
 
 	if s.InStandbyMode() {
 		s.logger.Info("Outage detected", "config threshold", outageThreshold, "time since last command", timeSinceLastCmd)
 		s.setMode(CommandMode)
+		s.logHandler.Append("Entered command mode", map[string]string{"timeSinceLastCmd": timeSinceLastCmd.String()})
 	}
 
 	currentInterval, err := s.planHandler.GetCurrentInterval(currentTime)
 	if err != nil {
+		s.logHandler.Append("No command available", nil)
```

Comment:
Questionable whether we want to log this every time - it could get noisy if the stored optimisation plan runs out (though that would only happen in a very long outage). I erred on the side of caution because otherwise it's hard to tell the difference between "log is quiet because everything is fine" vs "log is quiet because we're in an outage and have no commands to replay"

Reply:
Ideally, I think we should be conscious of the log file size and make appropriate decision when it exceeds some threshold, but that's perhaps a TODO for a different PR.

Reply:
Yeah, I feel if we will defer this, we will need to ensure to add this to the backlog. We must be able to have some form of log rotation. It can be done by other tools, not something we need to build here, but then, we need to plan for that as well.

Reply:
I would prefer any log rotation to happen outside the application code. We have an existing story for log management on the IPC, I'll add a task to that